### ISSUE TITLE
[Feature request] double/multi tap to cycle through special keys

### ISSUE THREAD

**Issue by iuuky** [futo-org/android-keyboard#123]
E.g. when I quickly type `uu` you would get `ú` and in case of `uuu` you would get `ů`. The specific final letter/symbol could be determined by the language selected. Timeout for these quick keys should be a separately configurable entry.

**Comment by abb128** [comment]
Are there any other keyboards that offer this as a feature? This would seem to conflict with fast typing

**Comment by iuuky** [comment]
Yes — [AnySoftKeyboard](https://github.com/AnySoftKeyboard/AnySoftKeyboard). I had to select a Slovak keyboard that had the description "Multitap for accent letter".

Not sure if it actually does as I'm not a fast writer.
